### DISCUSSION TITLE
Importing SortingState from @tanstack

### DISCUSSION THREAD

**Discussion by kpu23** [KevinVandy/material-react-table#345] [General]
The `type` SortingState is not exported with the `material-react-table` project. It needs to be imported directly from `@tanstack`.

I am confused why it's like that. I would expect it to be exported as part of this project so I don't need to reference a dependency that could change in the future.

Should I be importing the definition from `@tanstack`, or do you plan on exporting it from this project?

P.S. I am new to TS so please bear with me.

**Comment by KevinVandy** [comment]
Since I was not making any modifications to those types, I figured I leave them alone and just let them be exported from TanStack Table
